be/src/storage/persistent_index.cpp

```diff
@@ -2888,6 +2888,7 @@ Status PersistentIndex::on_commited() {
     if (_get_thread_pool != nullptr) {
         _get_thread_pool->shutdown();
     }
+    _get_thread_pool.reset();
```

Comment:
Can we use a global thread pool for this, instead of create and free thread pool every time.